reana_job_controller/config.py

```diff
@@ -19,9 +21,20 @@
 """Root path of the shared volume ."""
 
 JOB_BACKENDS = {
-    'kubernetes': KubernetesJobManager
+    'kubernetes': KubernetesJobManager,
+    'htcondorcern': HTCondorJobManagerCERN
 }
 """Supported job backends and corresponding management class."""
 
 DEFAULT_JOB_BACKEND = 'kubernetes'
 """Default compute backend for job submission."""
+
+HTCONDOR_SUBMISSION_JOB_IMG = os.getenv('HTCONDOR_SUBMISSION_JOB_IMG',
+                                        'batch-team/condorsubmit')
+"""Docker image to use for condor job submission."""
+
+HTCONDOR_SUBMITTER_POD_MAX_LIFETIME = 3600
```

Comment:
This should be definitely user-settable. Some jobs may need to run for more than that... IT would be good to expose this to users so that they can configure timeouts in `reana.yaml` for serial or in their workflow definition files (if supported).